library/src/main/java/com/getstream/sdk/chat/utils/CaptureController.java

```diff
@@ -0,0 +1,72 @@
+package com.getstream.sdk.chat.utils;
+
+import android.annotation.SuppressLint;
+import android.content.ContentValues;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Environment;
+import android.provider.MediaStore;
+
+import androidx.annotation.Nullable;
+
+import com.getstream.sdk.chat.R;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class CaptureController {
+
+    private static Uri imageUri, videoUri;
+
+    public static Intent getTakePictureIntent(Context context) {
+        File newFile = new File(getFolderPath(true, context), "IMG_" + getFileName() + ".jpg");
+        imageUri = Uri.fromFile(newFile);
+        Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
+        ContentValues values = new ContentValues();
+        values.put(MediaStore.Images.Media.TITLE, "New Picture");
```

Comment:
I assume these strings will be visible in system UI. If so they should be moved to `strings.xml`.